srv/components/misc_pkgs/ipmi/bmc_watchdog/install.sls

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+Install bmc-watchdog:
+  pkg.installed:
+    - name: freeipmi-bmc-watchdog
+
+Update watchdog timeout:
+  file.replace:
+    - name: /etc/sysconfig/bmc-watchdog
+    - pattern: '900'
```

Comment:
@ypise The entire file has only one line explaining different options. Since the optional parameter placing may vary, I'd checked the pattern directly for sleep time.
Ex: OPTIONS="-d -u 4 -p 0 -a 1 -F -P -L -S -O -i 300 -e 60"

Reply:
Can we not replace `-i 900` to `-i 300`?

Reply:
Done! Updated all the comments in the new commit.